README.md

```diff
@@ -96,7 +96,7 @@ Add `handleAuth()` to your app, which creates the following route handlers under
 
 ##### Add the Dynamic API Route
 
-Create a [catch-all, dynamic API route handler](https://nextjs.org/docs/api-routes/dynamic-api-routes#optional-catch-all-api-routes) under the `/pages/api` directory:
+Create a [catch-all, dynamic API route handler](https://nextjs.org/docs/pages/building-your-application/routing/api-routes#catch-all-api-routes) under the `/pages/api` directory:
```

Comment:
Should this link to dynamic routes given that it doesn't use the catchall (`...`) prefix? (I guess would need changing elsewhere too?)
```suggestion
Create a [dynamic API route handler](https://nextjs.org/docs/pages/building-your-application/routing/api-routes#dynamic-api-routes) under the `/pages/api` directory:
```